docker/start.sh

```diff
@@ -8,12 +8,13 @@ echo "Latest commit: " 1>&2
 cat /gitlog.txt 1>&2
 echo 1>&2
 
-# Change directory to the input directory. 
+# Change directory to the input directory.
 # Make it easier to pass gRPC service config relative to it, e.g.
 # --grpc-service-config google/cloud/texttospeech/v1/texttospeech_grpc_service_config.json
 
-cd /in 
+cd /in
 gapic-generator-typescript \
+  --gapic-validator_out=. \
```

Comment:
how is this option handled in `start_script.ts`?

Reply:
It will be passed to protoc command as `--gapic-validator_out=.` (added this logic)